.github/workflows/check-code-quality.yml

```diff
@@ -40,6 +40,6 @@ jobs:
         git diff origin/master... |
           (grep '^+' || (($? == 1)) ) |
           (grep -Eo '\b(https?)://[-A-Za-z0-9+&@#/%?=~_|$!:,.;]*[A-Za-z0-9+&@#/%=~_|$]' || (($? == 1)) ) |
-          sed 's/:/;/' |
```

Comment:
> But check-url.sh does expect a semicolon before the url to check.

Ahh, of course! 

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/f3e79f86dcd262c12664d862df78f38e8335242b/.github/workflows/check-url.sh#L7-L8

I guess for PRs, the filename will be `''`, then?

```console
dot|joshua@monarch:~$ filename=$(cut -d ";" -f 1 <<< ";test")
dot|joshua@monarch:~$ echo $filename
```

That's not too big a deal, I suppose, since most PRs will only ever add one or two URLs at most, meaning they won't be too hard to identify (vs. combing through 300+ links?)

Reply:
Oh! Good point, I didn't think that the filename was useful. We can add one!